arangod/Agency/Agent.cpp

```diff
@@ -263,7 +263,7 @@ priv_rpc_ret_t Agent::requestVote(term_t termOfPeer, std::string const& id,
 }
 
 /// Get copy of momentary configuration
-config_t const Agent::config() const { return _config; }
+config_t const& Agent::config() const { return _config; }
```

Comment:
The code comments isn't quite telling anymore, because the method now returns a reference, not a copy.

Reply:
Changed.